src/Decoder/EntryTransactionDetail.php

```diff
@@ -26,6 +26,15 @@ public function __construct(DateDecoderInterface $dateDecoder)
         $this->moneyFactory = new MoneyFactory();
     }
 
+    public function addCreditDebitIdentifier(DTO\EntryTransactionDetail $detail, SimpleXMLElement $CdtDbtInd): void
+    {
+        $creditDebitIdentifier = (string) $CdtDbtInd;
+        $creditDebitIdentifier = in_array($creditDebitIdentifier, ['CRDT', 'DBIT'], true)
```

Comment:
Have you considered using an Enum here ? IMHO we could drop support for PHP 8.0, if Enum make sense for this use-case. What do you think ?

Reply:
Generally I'm a fan of ENUM's. So no points against it. But it needs the raise first.
On the other hand .. more places could be benefit of enums. So maybe can be additional tasks.

Background is, that I need this change to solve a project need. Not only that, but I have to backport the whole lib afterwards to PHP5.6. And have quite some time pressure.

So if it would be okay, I would go with the current way. And we can make the PHP8.1+ raise then (new major/minor version?) and than make the enum using more throughout the places.